mint.json

```
@@ -27,7 +28,7 @@
   "topbarLinks": [
     {
       "name": "Support",
-      "url": "about/support"
+      "url": "https://www.ubicloud.com/docs/about/support"
```

This might also work

```suggestion
      "url": "/about/support"
```